docs/user/registering-microshift-devices-acm.md

```diff
@@ -0,0 +1,157 @@
+# Auto-Registering Devices with MicroShift into ACM
+
+## Configuring RHACM Agent Registration
+
+Auto-registration of devices with MicroShift into ACM relies on a feature called agent registration, allowing clusters to be imported via REST API calls using a CA bundle and token. Follow these [instructions](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#importing-managed-agent) to configure the necessary RBAC policies for your own user. Skip step 7 of the instructions in section 1.6.5.2.3, as this step will be automatically handled by the fleet devices.
+
+As part of this setup, you will need to collect the following information:
+
+- Agent registration URL
+- Token
+
+## Creating Repositories in Flight Control
+
+Flight Control includes a configuration provider that can retrieve data from an HTTP(S) endpoint. This integrates with ACM's agent registration API, requiring it to be formatted as a Flight Control repository.
+
+A sample YAML file for an HTTP repository looks like:
+
+```console
+apiVersion: v1alpha1
+kind: Repository
+metadata:
+  labels: {}
+  name: acm-test
+spec:
+  httpConfig:
+    token: $token
+  type: http
+  url: https://$agent_registration_host
+```
+
+Note that this repository will show as not valid, because the root URL does not return a value directly. However, this endpoint will be used in the fleet spec configuration section with a set of suffixes that will return the required Kubernetes manifests for auto-registration.
```

Comment:
Do we have a bug report for that? Can we make this an alert?
```
> [!NOTE] This repository will show ...
```

Reply:
I will highlight it with **_NOTE:_**

This is something I missed when designing the solution, that if you configure a root URL that doesn't return data it will be shown as not valid. I need to talk to @avishayt about this.